solutions/USACO open19-milk-factory.mdx

```diff
@@ -2,11 +2,59 @@
 id: usaco-940
 source: USACO Bronze 2019 December
 title: Milk Factory
-author: Mrinall Umasudhan and Jesse Choe
+author: Mrinall Umasudhan, Jesse Choe, and Ananth Kashyap
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_factory_bronze_open19.html)
 
+## Solution 1
+
+This solution follows the logic outlined in the official solution.
+
+## Implementation
+
+**Time Complexity**: $\mathcal{O}(N)$
+
+<LanguageSection>
+<PySection>
+
+```py
+# Take in the input using standard Python file i/o system
+data = open('factory.in').read().strip().split('\n')
+n = int(data[0])
+conveyer_belts = [list(map(int, data[i + 1].split(' '))) for i in range(n - 1)]
+
+# Create a list holding all factories that have an outgoing conveyer belts
+outgoing_locations = [0] * max(outgoing)
+
+for i in range(n - 1):
+	outgoing_locations[conveyer_belts[i][0] - 1] += 1
+
+ans = -1
+counter = 0
+
+# Find out which station does not have any outgoing conveyer belts
+for i in range(n - 1):
+	if outgoing_locations[conveyer_belts[i][1] - 1] == 0:
+		if conveyer_belts[i][1] != ans:
+			ans = conveyer_belts[i][1]
+			counter += 1
+
+# Make sure there is only one station with no outgoing belts
+if counter > 1:
+	ans = -1
+
+# Print the answer on the output file using Python file i/o system
```

Comment:
lines 22, 29, 35, and 47 are unnecessary

Reply:
I think it's fine to keep 29/35...

Reply:
the sections of code they divide are really part of the same code imo, so best to group them together